src/Quartz.AspNetCore/AspNetCore/HttpApi/Util/BadRequestException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Quartz.AspNetCore.HttpApi.Util;
+
+internal class BadRequestException : Exception
```

Comment:
nit: I think this is missing some constructors usually present with exceptions, also all internal classes should be `sealed` by default

Reply:
I only added the constructors that are needed at the moment. I will add the common constructors and mark these sealed.